### PR TITLE
Rename csand to csa

### PR DESCRIPTION
**Pull request by CAMOBAP795**
- #21 

Related PRs:
 - https://github.com/metanorma/metanorma.com/pull/196

TODO:
 - [x] ~Add backward compatibility tests~
 - [x] Fix hound issues

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,11 +1,12 @@
 = AsciiCSAND: Asciidoctor processor for CSA Normal Documents (CSAND)
 
-image:https://img.shields.io/gem/v/metanorma-csand.svg["Gem Version", link="https://rubygems.org/gems/metanorma-csand"]
-image:https://travis-ci.com/metanorma/metanorma-csand.svg["Build Status", link="https://travis-ci.com/metanorma/metanorma-csand"]
-image:https://ci.appveyor.com/api/projects/status/gs07fuoo1xhe1spe?svg=true["Appveyor Build Status", link="https://ci.appveyor.com/project/metanorma/metanorma-csand"]
+image:https://img.shields.io/gem/v/metanorma-csa.svg["Gem Version", link="https://rubygems.org/gems/metanorma-csa"]
```
Comment (CAMOBAP795) [inline README.adoc]: @ronaldtse @opoudjis probably we should not rename this till actual release, please let me know how do you think

File: README.adoc [hunk 2]
```diff
@@ -71,12 +72,12 @@ scripts such as this one. You need only run the following in a Terminal console:
...
 $ bash <(curl -s https://raw.githubusercontent.com/metanorma/metanorma-macos-setup/master/metanorma-setup)
-$ gem install metanorma-csand
+$ gem install metanorma-csa
 
 ----
 
 == Documentation
 
-See https://www.metanorma.com/author/csand/[Author Cloud Security Alliance documents with Metanorma-CSAND].
+See https://www.metanorma.com/author/csa/[Author Cloud Security Alliance documents with Metanorma-CSAND].
```
Comment (CAMOBAP795) [inline README.adoc]: @ronaldtse will we rename this in future or I should revert this change?

File: lib/asciidoctor/csa/converter.rb [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
+require "isodoc/csa/word_convert"
 require "asciidoctor/standoc/converter"
 require "fileutils"
 require_relative "./validate.rb"
 
 module Asciidoctor
-  module Csand
-        CSAND_NAMESPACE = "https://open.ribose.com/standards/csand"
+  module Csa
+    CSA_NAMESPACE = "https://open.ribose.com/standards/csand"
```
Comment (CAMOBAP795) [inline lib/asciidoctor/csa/converter.rb]: @ronaldtse @opoudjis I didn;t changed XML namespace to save backward compatibility, make sense?

File: lib/asciidoctor/csa/converter.rb [hunk 4]
```diff
@@ -77,15 +80,15 @@ def title_validate(root)
       end
 
       def makexml(node)
-        result = ["<?xml version='1.0' encoding='UTF-8'?>\n<csand-standard>"]
+        result = ["<?xml version='1.0' encoding='UTF-8'?>\n<csa-standard>"]
         @draft = node.attributes.has_key?("draft")
         result << noko { |ixml| front node, ixml }
         result << noko { |ixml| middle node, ixml }
-        result << "</csand-standard>"
+        result << "</csa-standard>"
```
Comment (CAMOBAP795) [inline lib/asciidoctor/csa/converter.rb]: @ronaldtse @opoudjis let me know if this change should not be done, BTW I updated XML grammar to allow both names `csand-standard` and `csa-standard` (for backward compatibility)

File: lib/asciidoctor/csa/csa.rng [hunk 5]
```diff
@@ -79,8 +79,13 @@
...
   </include>
-  <define name="csand-standard">
-    <element name="csand-standard">
+  <define name="csa-standard">
+    <element>
+      <choice>
+        <!-- https://stackoverflow.com/a/49798676/902217 -->
+        <name>csand-standard</name>
+        <name>csa-standard</name>
+      </choice>
```
Comment (CAMOBAP795) [inline lib/asciidoctor/csa/csa.rng]: @ronaldtse @opoudjis we support both names, no backward compatibility tests added yet, but I will try to add someting

**Comment by ronaldtse** [comment]
@CAMOBAP795 I don't think we need backward compatibility since the users are minimal. Let's just go with the new name.

File: lib/asciidoctor/csa/converter.rb (hunk 4 above)
Comment (ronaldtse) [reply lib/asciidoctor/csa/converter.rb]: @CAMOBAP795 let's change it.

File: lib/asciidoctor/csa/converter.rb (hunk 3 above)
Comment (ronaldtse) [reply lib/asciidoctor/csa/converter.rb]: @CAMOBAP795 let's change it as well.

File: README.adoc (hunk 2 above)
Comment (ronaldtse) [reply README.adoc]: Yes let's rename it. (please also make a PR in metanorma.com)

File: README.adoc (hunk 1 above)
Comment (ronaldtse) [reply README.adoc]: @CAMOBAP795 let's change it.

File: lib/asciidoctor/csa/csa.rng (hunk 5 above)
Comment (ronaldtse) [reply lib/asciidoctor/csa/csa.rng]: The gem isn't in popular use, let's change it without backward compatibility.

File: README.adoc (hunk 2 above)
Comment (CAMOBAP795) [reply README.adoc]: @ronaldtse https://github.com/metanorma/metanorma.com/pull/196

**Comment by CAMOBAP795** [comment]
I tried to fix as much lint errors as possible,but there is still some left

**Comment by CAMOBAP795** [comment]
I'm going to merge it

**Pull request merged**